__tests__/buildTranslationFiles.spec.ts

```diff
@@ -259,20 +262,12 @@ describe.each(formats)('buildTranslationFiles in %s', (fileFormat) => {
               end: 2,
               prefix: 'site-header.navigation.route.nested',
             }),
-          },
-          todos: {
-            ...generateKeys({ end: 2, prefix: 'numbers' }),
+            ...generateKeys({ end: 2, prefix: 'todosPage.numbers' }),
           },
         };
 
         createTranslations(config);
         assertTranslation({ type, expected: expected.global, fileFormat });
-        assertTranslation({
```

Comment:
Can you share why this was removed?

Reply:
See https://github.com/jsverse/transloco-keys-manager/pull/177#discussion_r1494806657